mne_bids/read.py

```diff
@@ -774,19 +779,27 @@ def get_head_mri_trans(bids_path, extra_params=None, t1_bids_path=None):
     # The MRI landmarks are in "voxels". We need to convert the to the
     # neuromag RAS coordinate system in order to compare the with MEG landmarks
     # see also: `mne_bids.write.write_anat`
-    t1w_path = t1w_json_path.replace('.json', '.nii')
-    if not op.exists(t1w_path):
-        t1w_path += '.gz'  # perhaps it is .nii.gz? ... else raise an error
-    if not op.exists(t1w_path):
-        raise RuntimeError('Could not find the T1 weighted MRI associated '
-                           'with "{}". Tried: "{}" but it does not exist.'
-                           .format(t1w_json_path, t1w_path))
-    t1_nifti = nib.load(t1w_path)
+    fs_subject = meg_bids_path.subject if fs_subject is None else fs_subject
```

Comment:
```suggestion
    fs_subject = ("sub-" + meg_bids_path.subject) if fs_subject is None else fs_subject
```
I think it's correct